src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (inversedRobot[value]) {
```

Comment:
It'd be better to use `value in` or `hasOwnProperty()` to check if the key exists in the object. You got lucky that we don't have values like: `false`, `0` or empty string in this task, but your check wouldn't work properly with such values.

Reply:
`hasOwnProperty()` wouldn't pass the linter, so I guess using `in` operator is the right answer

Reply:
I guess linter ignores me..because hasOwnProperty() passed